CONTRIBUTING.rst

```diff
@@ -272,15 +272,16 @@ following exceptions:
   comment, ``//``, is not allowed.
 * Use ``/**  */`` for doxygen comments that need to appear in the documentation.
 
-The Linux kernel GPL-licensed tool ``checkpatch`` is used to check coding
-style conformity. Checkpatch is available in the scripts directory. To invoke
-it when committing code, edit your *.git/hooks/pre-commit* file to contain:
+The Linux kernel GPL-licensed tool ``checkpatch`` is used to check
+coding style conformity. Checkpatch is available in the scripts
+directory. To invoke it when committing code, make the file
+*$ZEPHYR_BASE/.git/hooks/pre-commit* executable and edit it to contain:
 
 .. code-block:: bash
 
     #!/bin/sh
     set -e exec
-    exec git diff --cached | ${ZEPHYR_BASE}/scripts/checkpatch.pl - || true
+    exec git diff --cached | scripts/checkpatch.pl -
```

Comment:
this will not work if you commit from a sub-directory.

Reply:
Could have sworn I tested this, will double-check.

Reply:
Yeah, works fine, evidently, git hooks are invoked from the root directory.

Reply:
well, I remember very vividly this used not to work, maybe that was with an older version of git.

Reply:
I'll put the env var back in then, I can't prove it works for all environments.

Reply:
Fixed.